### PR TITLE
[Auto Bench] Scripts of Running Benchmarks with Nginx

### LINKED ISSUE #2826: [Auto Benchmarks] - Use nginx in scripts of running benchmarks in AWS
### What is this task and why do we need to work on it?

Current benchmarking scripts doesn't use nginx, however this is needed when `total_nodes >= 500` and `transaction_size` is large.

### What work will need to be done to complete this task?

- [x] Use nginx in scripts of running benchmarks in AWS

### What are the acceptance criteria to close this issue?

After this , the script should be able to run with `total_nodes \in {500, 1000}`, `transaction_size \in {1000000 20000000}`.

### Branch work will be merged to (if not the default branch)

_No response_

### PR DESCRIPTION
**Pull request by dailinsubjam**
Closes #2826 .

### This PR: 
- Adds the script for benchmarking when using nginx.
- Raise up runnable tx size to 1e7 bytes. With 50 nodes and 5 da committee it can get 2.37MBps. With tx size of 2e8 and 10/5 nodes, it can get 15.56MBps. More results in `results_init_run.csv`.
- Supports loop-over-tuples-in-bash for parameter assignment.
- Changed some `debug` and `error` log to `println`, since our Datadog does not support `info`(not sure why) and `debug`, and it takes `println` as `info`. 

### This PR does not: 
- Guarantee large total nodes (> 100) and large tx size (> 1e7 bytes) could finish runs successfully, I still meet lots of time outs using nginx.

### Key places to review: 
`scripts/aws_ecs_nginx_benchmarks.sh`

### How to test this PR: 
Follow the testing steps here: https://github.com/EspressoSystems/HotShot/pull/2793 .
And change the script to `./script/aws_ecs_nginx_benchmarks.sh`.
Remember to run nginx following the step mentioned in the script.

### CODE REVIEW

File: crates/examples/infra/mod.rs [hunk 1]
```diff
@@ -570,10 +570,10 @@ pub trait RunDA<
         let mut total_latency = 0;
         let mut num_latency = 0;
 
-        debug!("Sleeping for {start_delay_seconds} seconds before starting hotshot!");
+        println!("Sleeping for {start_delay_seconds} seconds before starting hotshot!");
```
Comment (jparr721) [inline crates/examples/infra/mod.rs:573]: Looks like this PR 2858 have similar changes. Overall I think removing logging for print statements will make parsing logs for us more difficult when not in datadog (which is a big part of the workflow). Perhaps we can just set datadog to take info logs in docker and then filtein the UI, or filter the executable as `RUST_LOG=warn,hotshot_orchestrator::config=info` per Salman's suggestion?

**Review by jparr721 — CHANGES_REQUESTED** [review]
It might be worth a team discussion about these logs.

File: crates/examples/infra/mod.rs (hunk 1 above)
Comment (dailinsubjam) [reply crates/examples/infra/mod.rs:573]: Yes, this PR is covered by that one. I set all of these to `info` in that PR, and I'm using `RUST_LOG=warn,hotshot_orchestrator::config=info` locally.

**Comment by dailinsubjam** [comment]
Coverd by https://github.com/EspressoSystems/HotShot/pull/2858

**Pull request closed without merging**